Beta

st0rmtr0oper opened the pull request:
Работает фрагмент с афишой, данные загружаются

Commit:
продолжение следует...
почему то не работает .getTodayFilms() в PosterFragment

MrFiring commented on app/build.gradle.kts:
```diff
@@ -63,12 +65,12 @@ dependencies {
     androidTestImplementation("androidx.test.espresso:espresso-core:3.5.1")
 
     //retrofit
+    implementation("com.jakewharton.retrofit:retrofit2-kotlin-coroutines-adapter:0.9.2")
```
Эта штука не нужна, ретрофит из коробки с корутинами умеет работать)

Commit:
исправлена ошибка

MrFiring commented on app/src/main/java/com/example/cinemashift/data/CustomDataClasses.kt:
```diff
@@ -13,14 +13,13 @@ data class FilmsResponse(
     val description: String,
     val releaseDate: String,
-    val actors: ArrayList<FilmPerson>,
+//    val actors: ArrayList<FilmPerson>,
```
ArrayList это что то на Java) В котлине можно просто List<T> использовать.

Commit:
теперь уже и в сеть ходит.
пока просто кидает json в текст ошибки. ща допилим

Commit:
окончательная замена moshi на gson, с ним куча заморочек

Commit:
небольшие изменения, PosterAdapter bind

Commit:
оно ожило!
теперь подгрузка афиши полностью работает. очень некрасиво, правда, но это детали
